atomic_reactor/rpm_util.py

```diff
@@ -0,0 +1,89 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+image_component_rpm_tags = [
+    'NAME',
+    'VERSION',
+    'RELEASE',
+    'ARCH',
+    'EPOCH',
+    'SIZE',
+    'SIGMD5',
+    'BUILDTIME',
+    'SIGPGP:pgpsig',
+    'SIGGPG:pgpsig',
+]
+
+
+def rpm_qf_args(tags=image_component_rpm_tags, separator=';'):
+    """
+    Return the arguments to pass to rpm to list RPMs in the format expected
+    by parse_rpm_output.
+    """
+
+    fmt = separator.join(["%%{%s}" % tag for tag in image_component_rpm_tags])
+    return r"-qa --qf '{0}\n'".format(fmt)
+
+
+def parse_rpm_output(output, tags=image_component_rpm_tags, separator=';'):
```

Comment:
Landscape.io complains that tags takes a dangerous default value because it is mutable.